bench/irmin-pack/dune

```diff
@@ -1,8 +1,10 @@
 (executables
  (names main layers)
+ (public_names bench-pack bench-pack-layers)
+ (package irmin-bench)
  (preprocess
   (pps ppx_deriving_yojson ppx_repr))
- (libraries irmin-pack irmin-test.bench irmin-layers lwt unix cmdliner logs
+ (libraries irmin-pack irmin-test.bench irmin-layers lwt cmdliner logs
```

Comment:
That one is unfortunate: pretty sure it's needed by `(implicit_transitive_deps false)`, so this is a bit of a step backwards.

Reply:
I've reverted that change